PendingReleaseNotes

```diff
@@ -21,6 +21,8 @@
 * RGW: Adding missing quotes to the ETag values returned by S3 CopyPart,
   PostObject and CompleteMultipartUpload responses.
 * RGW: Added support for S3 GetObjectAttributes.
+* RGW: Added beast frontend option so_reuseport which facilitates running multiple
+       RGWs processes on same host to share the same port number.
```

Comment:
Beast should be capitalized.  Should `so_reuseport` here be uppercase, or does the frontends value want it lowercase?

also:

>running multiple RGW instances on the same host by sharing a single TCP port

Reply:
done. `so_reuseport` needs to be lowercase as required by beast